custom/icds_reports/templates/icds_reports/dashboard.html

```diff
@@ -58,8 +58,47 @@
 {% endcomment %}
         <script src="{% static 'js/topojsons/states_v2.topojson.js' %}"></script>
         <script src="{% static 'js/topojsons/districts_v2.topojson.js' %}"></script>
+
+        {# to be removed once  once we move the topojson change from feature flag #}
         <script src="{% static 'js/topojsons/blocks_v3.topojson.js' %}"></script>
 
+        <script src="{% static 'js/topojsons/blocks/maharashtra_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/madhya_pradesh_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/telangana_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/andhra_pradesh_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/arunachal_pradesh_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/tamil_nadu_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/meghalaya_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/assam_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/punjab_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/odisha_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/gujarat_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/himachal_pradesh_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/uttar_pradesh_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/nagaland_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/haryana_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/jharkhand_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/uttarakhand_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/bihar_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/west_bengal_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/j&k_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/chhattisgarh_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/rajasthan_blocks_v3.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/blocks/andaman_&_nicobar_islands_blocks_v3.topojson.js' %}"></script>
```

Comment:
it may work because of magic auto-encoding, but it'd certainly be safer to remove the `&` character from the filenames here

Reply:
@czue , Along with removing '&' from file name I'll also fix the test case which is breaking, please don't approve/merge the branch till then.